work11/index.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+const div = document.querySelector(".wrap")
+const ul = document.getElementById("js-list");
+const url = "https://jsondata.okiba.me/v1/json/d4ex5210822124352"
+
+function getData() {
+  const result = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(
+        async function () {
+          const response = (await fetch(url)).json();
+          return response
+        }()
+      )
+
+    }, 3000);
+  })
+  return result;
+}
+
+async function displayView() {
+  loading();
+
+  try {
+    const arrayData = await getData();
+    createElement(arrayData);
+  } catch (e) {
+    div.textContent = e.message;
+  } finally {
+    hideLoading();
+  }
+
+}
+
+function hideLoading() {
+  ul.style.backgroundImage = "none";
+}
+
+function createElement(imgArray) {
```

Comment:
`imgArray `
This is not appropriate because the image array is not being passed around.
`responseData` or `data`  I think this is a good idea.